preprocessing.py

```diff
@@ -0,0 +1,102 @@
+import torch  
+import numpy as np
+from transformers import BertTokenizer,BertModel
```

Comment:
```suggestion
from transformers import BertTokenizer
```
The BERT model is instantiated and used in the model.py